.github/workflows/playwright.yml

```diff
@@ -1,43 +1,61 @@
 name: Playwright Tests
 
-on: [push, pull_request]
+on: workflow_call
 
 jobs:
-  test:
+  acceptance:
     strategy:
+      fail-fast: false
```

Comment:
What does this do? Keep running other tests in the matrix when one fails?

Reply:
Yeah, otherwise we have scenarios like the linux ones failing (due to missing snapshots) and preventing the windows actuons from uploading their snapshots